.github/workflows/release_master_fontpatcher-zip.yaml

```diff
@@ -0,0 +1,45 @@
+name: Release font patcher from master
+
+on:
+  create:
```

Comment:
Not sure why we want to trigger on `create`?

Reply:
That was for my own testing, forgive me.